src/components/CheckableButton/CheckableButton.scss

```diff
@@ -78,14 +68,9 @@ $chekbox-label-margin: rem(20px);
   }
 
   &.CheckableButton-plain {
-    border: border(transparent);
```

Comment:
should this be `border: none` so we override the border set on `.CheckableButton`?